apps/users/mail.py

```diff
@@ -17,15 +17,15 @@ def create_confirm_account_email(profile: Profile) -> Mail:
             "token": account_activation_token.make_token(profile.user),
         },
     )
-    email = Mail(
+    email = EmailMessage(
         subject=mail_subject,
-        html_content=message,
+        body=message,
         from_email="noreply@cosmostue.nl",
-        to_emails=profile.user.username,
+        to=[profile.user.username],
         # List-Unsubscribe headers allows for easy unsubscribing from the mailing list by sending an email to
         # the specified email automatically. Is needed to not be classified as spam, and very import to handle
         # any requests correctly.
         # headers={"List-Unsubscribe": "mailto:webcom@cosmostue.nl"},
```

Comment:
Previously this required the list unsubscribe header to be added manually. Does the sendgrid smtp service add it automatically or do we need to uncomment this again now?

Reply:
Reset emails don't add it, so it is probably fine to not add it here as well. If we have problems in the future adding it in should be easy.